Hymns/Tags/SelectLabelView.swift

```diff
@@ -0,0 +1,107 @@
+import SwiftUI
+
+struct SelectLabelView: View {
```

Comment:
`ColorSelectorView` makes more sense

Reply:
fixed for you. feel free to resolve if it makes sense.

Reply:
Looks good!